tests/test_volume.py

```diff
@@ -813,6 +813,10 @@ def test_transformation_symmetry_warnings(symmetric_vols):
     assert str(vol_c3.symmetry_group) == "C3"
 
 
+@pytest.mark.skipif(
```

Comment:
I'm still not sure I understand this situation. Can you explain it again? Thanks

Reply:
Sure. Simply adding the `--strict-warnings` flag to workflow line `python -m pytest --durations=50 --strict-warnings` resulted in the error:
```
ERROR: usage: __main__.py [options] [file_or_dir] [file_or_dir] [...]
__main__.py: error: unrecognized arguments: --strict-warnings
  inifile: /var/ci/_work/ASPIRE-Python/ASPIRE-Python/tox.ini
  rootdir: /var/ci/_work/ASPIRE-Python/ASPIRE-Python
```

I used the workaround of handling the errors with an environment variable like so:
```
PYTHONWARNINGS=error python -m pytest --durations=50
```
This caused us to pick up cached warnings causing this test to fail.

Now that I'm looking at the error message again it might be the case that we need to edit the `tox.ini` file to let pytest know we want to use `--strict-warnings`. I'm not sure why pytest recognizes the `--durations` flag but not the `--strict-warnings` flag.

Reply:
It's the `pick up cached warnings ` that I don't understand.  What warnings? Where/why are they cached?

Reply:
The warnings are coming from line 856 which is in a loop. We are testing that if the same line emits a warning more than one time that only one warning is emitted. Python caches all of the warnings in a warnings registry but only emits a single warning per source location. Due to using the environment variable as opposed to pytest to catch the warnings, all of the warnings in the warnings registry are being reported as opposed to just the ones that would be passed to pytest.

After thinking about this, I'm not sure we even need to be testing this behavior. This is a default behavior for python that only changes if we specifically change it, for example, by setting `warnings.simplefilter("always")`.

Reply:
Okay, think I got it.  Yeah, it might just be easier to remove this problem case then :)

Reply:
Done. Thanks!